opwen_email_server/mailers/shared.py

```diff
@@ -0,0 +1,47 @@
+from typing import Callable
+from typing import Dict
+
+from libcloud.storage.types import ObjectDoesNotExistError
+
+from opwen_email_server.actions import _Action
+from opwen_email_server.services.storage import AzureObjectStorage
+from opwen_email_server.services.storage import AzureTextStorage
+from opwen_email_server.utils.email_parser import MimeEmailParser
+from opwen_email_server.utils.unique import new_email_id
+
+
+class ProcessServiceEmail(_Action):
```

Comment:
Would also be good to add a unit test for this. See examples in [test_actions.py](https://github.com/ascoderu/opwen-cloudserver/blob/43671478d9fd40ce21bf0143875a9f1fe1f4070e/tests/opwen_email_server/test_actions.py).

Reply:
👍